packages/gnarly-reducer-erc20/src/reducer.ts

```diff
@@ -15,61 +15,80 @@ import {
 import abi from './abi'
 
 const TRANSFER_REASON = 'ERC20_TRANSFER'
+const MINT_REASON = 'ERC20_MINT'
+const BURN_REASON = 'ERC20_BURN'
+
+const isZeroAddress = (address: string) => address === '0x0000000000000000000000000000000000000000'
```

Comment:
probably there is a better way to do this, ideas?

Reply:
I expected there would be an `isNullAddress` utility in web3, but apparently there isn't. Let's leave this here for now, but I've added issue #39 to track it